benefits/locale/es/LC_MESSAGES/django.po

```diff
@@ -259,8 +259,8 @@ msgstr ""
 "califican para una serie de programas de tarifas reducidas. Este beneficio "
 "de tránsito debe renovarse en el futuro en función de la fecha de "
 "vencimiento de la tarjeta de cortesía. Obtenga más información en el <a "
-"href='https://mst.org/riders-guide/how-to-ride/courtesy-card/' "
-"target=\"_blank\" rel=\"noopener noreferrer\">MST Riders Guide</a>."
+"href='https://mst.org/riders-guide/how-to-ride/courtesy-card/' target="
+"\"_blank\" rel=\"noopener noreferrer\">MST Riders Guide</a>."
```

Comment:
Django loves to move this every other pull request.

Reply:
I tend to avoid committing this change if possible.